pages/story/story_data.js

```diff
@@ -0,0 +1,28 @@
+
+//write functions here to take data and push to story_array
+
+export const story_array = 
+[{
+    num : 1,
+    href: "/story/1", //keep this unique
+    teampic: "https://images.unsplash.com/photo-1628891890467-b79f2c8ba9dc?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8c3BvcnRzJTIwdGVhbXxlbnwwfHwwfHw%3D",
```

Comment:
Lets follow camel case everywhere. so here it will be `teamPic`

Reply:
generally I write `snake_case`, made its way into this too :), will change to camelcase.